dockerfiles/centos_7

```diff
@@ -7,17 +7,18 @@ ARG TNT_VER \
 
 ENV GC64=${GC64:-false} \
     TARANTOOL_INSTALLER_URL=https://www.tarantool.io/release/${TNT_VER:-2}/installer.sh \
-    LUAROCK_VSHARD_VERSION=0.1.18 \
+    LUAROCK_VSHARD_VERSION=0.1.20 \
     LUAROCK_AVRO_SCHEMA_VERSION=3.0.6 \
     LUAROCK_EXPERATIOND_VERSION=1.1.1 \
-    LUAROCK_QUEUE_VERSION=1.1.0 \
+    LUAROCK_QUEUE_VERSION=1.2.0 \
     LUAROCK_CONNPOOL_VERSION=1.1.1 \
-    LUAROCK_HTTP_VERSION=1.2.0 \
+    LUAROCK_HTTP_VERSION=1.3.0 \
     LUAROCK_MEMCACHED_VERSION=1.0.1 \
-    LUAROCK_METRICS_VERSION=0.12.0 \
+    LUAROCK_METRICS_VERSION=0.14.0 \
     LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
-    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.0 \
-    LUAROCK_TARANTOOL_GIS_VERSION=1.0.0 \
+    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.1 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.1 \
```

Comment:
As far as I know we shouldn't add mqtt to CentOS. https://github.com/tarantool/docker/pull/58

Reply:
done